src/main/java/com/linkmoa/domain/member/entity/Member.java

```diff
@@ -31,10 +40,36 @@ public class Member {
     @Enumerated(EnumType.STRING)
     private Provider provider;
 
+    @Column(name="password")
+    private String password;
+
+    @Column(name = "member_age_range")
+    private String ageRange;
+
+    @Column(name="member_gender")
+    private Gender gender;
+
+    @Column(name = "member_job")
+    private String job;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name="role")
+    private Role role;
+
+    @OneToMany(
+        mappedBy = "member",
+        cascade = CascadeType.ALL,
+        orphanRemoval = true
+    )
+    private List<MemberPageLink> memberPageLinkList= new ArrayList<>();
+
     @Builder
-    public Member(String email, String nickname, Provider provider) {
+    public Member(String email, String nickname, Provider provider,String password,Role role) {
         this.email = email;
         this.nickname = nickname;
         this.provider = provider;
+        this.password=password;
+        this.role=role;
     }
```

Comment:
Nit: 이것도 보통 린터에서 걸리는 내용 같지만, 저희가 이미 팀 컨벤션이 있어서 가독성을 위해 공백을 추가해주시면 좋을 것 같습니다.

- [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html)

```md
4.6.2 Horizontal whitespace
Beyond where required by the language or other style rules, and apart from literals, comments and Javadoc, a single ASCII space also appears in the following places only.
...
- On both sides of any binary or ternary operator. This also applies to the following "operator-like" symbols:
...
- After ,:; or the closing parenthesis ()) of a cast
```

```suggestion
    public Member(String email, String nickname, Provider provider, String password, Role role) {
        this.email = email;
        this.nickname = nickname;
        this.provider = provider;
        this.password = password;
        this.role = role;
    }
```